### PR TITLE
fix: changed menu-dropdown content style to body-1-regular

### PR DESCRIPTION
**Pull request by Greedy-Geek**

### CODE REVIEW

File: projects/components/src/menu-dropdown/menu-dropdown.component.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 @import 'interaction';
```
Comment (itssharmasandeep) [inline projects/components/src/menu-dropdown/menu-dropdown.component.scss:1]: Please remove this. `mixins` covers this

File: projects/components/src/menu-dropdown/menu-dropdown.component.scss [hunk 2]
```diff
@@ -35,5 +36,5 @@
 }
 
 .dropdown-content {
-  @include popover-menu-content;
+  @include body-1-regular;
```
Comment (itssharmasandeep) [inline projects/components/src/menu-dropdown/menu-dropdown.component.scss:39]: is this the change we really want?
These are two completely different styles.
cc: @arjunlalb

Comment (arjunlalb) [reply projects/components/src/menu-dropdown/menu-dropdown.component.scss:39]: We need both here. We need the popover styling + a styling for the text.

@Greedy-Geek do not remove the existing container styling. Also, please test the changes before sending out a PR.

Comment (Greedy-Geek) [reply projects/components/src/menu-dropdown/menu-dropdown.component.scss:39]: Oh okay, I checked the changes before sending for the PR; maybe I missed some obvious things.

File: projects/components/src/menu-dropdown/menu-dropdown.component.scss (hunk 1 above)
Comment (Greedy-Geek) [reply projects/components/src/menu-dropdown/menu-dropdown.component.scss:1]: Okay.

**Pull request closed without merging**